shell/app/config-page/components/table/table.tsx

```diff
@@ -290,7 +285,7 @@ const BatchOperation = (props: IBatchProps) => {
       <span className="mr-2">{`${i18n.t('selected {xx}', {
         xx: `${selectedRowKeys?.length || 0}${i18n.t('common:items')}`,
       })}`}</span>
-      <Dropdown overlay={dropdownMenu}>
+      <Dropdown overlay={dropdownMenu} zIndex={1000}>
```

Comment:
is `zIndex` a Dropdown api?

Reply:
yes, `export interface PopconfirmProps extends AbstractTooltipProps`

Reply:
yes, `export interface PopconfirmProps extends AbstractTooltipProps`